tests/e2e.sh

```diff
@@ -68,7 +68,7 @@ touch "${HOME}"/.kube/config
 export KUBECONFIG=$HOME/.kube/config
 
 if [[ "$MINIKUBE_DRIVER" != "none" ]]; then 
-  export MINIKUBE_PROFILE_ARG="--profile ${MINIKUBE_PROFILE}"
+  export MINIKUBE_PROFILE_ARG=" --profile ${MINIKUBE_PROFILE}"
```

Comment:
I believe you can solve this issue by setting the variable the following way:
```suggestion
  export MINIKUBE_PROFILE_ARG="--profile=${MINIKUBE_PROFILE}"
```
What happened previously was that the variable was replaced literally by: `'--profile ksm-e2e'` since it contains space separated strings.

Note: adding a space before the profile option here will actually have a really different behaviour from what you expect. Bash will process it the following way: ```' --profile ksm-e2e'```. Since this option is not starting by either `-` or `--` minikube will from my understanding skip it. In order to test this behaviour, you can set `MINIKUBE_PROFILE_ARG` to `this option will be skipped`.

Reply:
Maybe the best option then is to make the profile itself the variable and always set it? How would minikube react to an empty string for that flag?

Reply:
I don't know, my guess would be that they replace it by the [default value](https://github.com/kubernetes/minikube/blob/4f704271d7f8802bba7ec6c72b80bba2e4d4f021/site/content/en/docs/commands/config.md#options-inherited-from-parent-commands-5). Otherwise, we can set it to its default value `minikube`. What do you think?

Reply:
Sounds good to me. Could you do a PR to change the strategy to that?

Reply:
Sure